client/consensus/aura/src/lib.rs

```diff
@@ -654,9 +660,9 @@ fn register_aura_inherent_data_provider(
 	inherent_data_providers: &InherentDataProviders,
 	slot_duration: u64,
 ) -> Result<(), consensus_common::Error> {
-	if !inherent_data_providers.has_provider(&paint_aura::INHERENT_IDENTIFIER) {
+	if !inherent_data_providers.has_provider(&aura_primitives::inherents::INHERENT_IDENTIFIER) {
```

Comment:
Can't we just import it directly? I find it a bit confusing to have both `use aura_primitives::...` at the top and then use full path to import some other stuff (I know the other one is `pub use`, but still the full path here feels a bit too long).